# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1426126907 14400
#      Wed Mar 11 22:21:47 2015 -0400
# Node ID ebdecd5cd8c75cdcb9848b8e02ef9a5074e0ffbb
# Parent  3c414ad460d5278d3f5ace5be9227c32055e879d
Restore blank line between docstring and additional help text

* __additional_help_message__.m: Insert newline at the start of the
additional help message when printed.  This restores one line of padding
out of the two that were removed with cset e884d5816471.

diff --git a/scripts/help/private/__additional_help_message__.m b/scripts/help/private/__additional_help_message__.m
--- a/scripts/help/private/__additional_help_message__.m
+++ b/scripts/help/private/__additional_help_message__.m
@@ -21,17 +21,17 @@
 ## Undocumented internal function.
 ## @end deftypefn
 
 function msg = __additional_help_message__ ()
 
   if (suppress_verbose_help_message ())
     msg = "";
   else
-    msg = "\
+    msg = "\n\
 Additional help for built-in functions and operators is\n\
 available in the online version of the manual.  Use the command\n\
 'doc <topic>' to search the manual index.\n\
 \n\
 Help and information about Octave is also available on the WWW\n\
 at http://www.octave.org and via the help@octave.org\n\
 mailing list.\n";
   endif
